docs/README.md

```diff
@@ -10,54 +10,47 @@
-| [Admin Customers](https://developers.vtex.com/vtex-developer-docs/docs/vtex-admin-customers) | Provides additional customer data management capabilities on the VTEX Admin, allowing store administrators to create, edit and search for customers. |
+| - | - |
+| [Storefront Permissions](https://developers.vtex.com/docs/guides/vtex-storefront-permissions) | Stores a predefined set of roles and permissions related to what B2B users can access and do in the storefront, which other integrated apps can check. This is useful for stores that want to set specific permissions for users with different organizational roles. |
+| [Storefront Permissions UI](https://developers.vtex.com/docs/guides/vtex-storefront-permissions-ui) | Provides an interface for the [Storefront Permissions](https://developers.vtex.com/docs/guides/vtex-storefront-permissions) app, communicates with its stored roles and permissions, and provides the following features for this scenario. <ul><li><b>Role management:</b> Allows VTEX Admin users to manage B2B roles and associated app permissions through an interface.</li>  <li><b>Theme block configuration:</b> Enables conditional theme blocks so only users with the required permissions can access specific content in your storefront.</li></ul> |
+| [B2B Organizations](https://developers.vtex.com/docs/guides/vtex-b2b-organizations) | Enables you to group B2B users into organizations to which you can assign specific payment methods, price tables, and product collections. This allows all organization users to share the same commercial conditions. Each organization is further segmented into one or more cost centers, with its own shipping addresses, which will be available for cost center users at checkout. |
+| [B2B Quotes](https://developers.vtex.com/docs/guides/vtex-b2b-quotes) | Enables B2B users to create quotes and saved carts, which are then shared with the other members of their organization. Users with the appropriate storefront permissions can modify quotes to apply discounts or change the amount of items, so the resulting quote can then be used to place an order. |
+| [B2B Checkout Settings](https://developers.vtex.com/docs/guides/vtex-b2b-checkout-settings) | Extends the checkout experience for users who are members of B2B organizations. With this app, you can allow specific payment methods for the user's organization, provide prefilled addresses based on the cost center of each user, add a purchase order number to the order, and allow customers to create an order quote before placing the order. |
+| [B2B Orders History](https://github.com/vtex-apps/b2b-orders-history) | Provides a new **Orders** page under **My Account** so that B2B users can view all previous orders placed by users in their organization or cost center, depending on their role. |
```

Comment:
```suggestion
| [B2B Orders History](https://developers.vtex.com/docs/apps/vtex.b2b-orders-history) | Provides a new **Orders** page under **My Account** so that B2B users can view all previous orders placed by users in their organization or cost center, depending on their role. |
```